### PR TITLE
Tool 343 analytics trkref filters

### PR DESCRIPTION
**Pull request by seddy**
The trkrefs filter from Tableau cannot be used to populate the list of trkrefs because it will limit the number of trkrefs to 200 and it will not contain the Retailer name. Instead, we can use the list of trkrefs that are coming back from the /profile request done to client_portal-admin.

### CODE REVIEW

File: app/js/services/cp_analytics_api_client.js [hunk 1]
```diff
@@ -2,3 +2,4 @@ import { CP_ANALYTICS_API } from '../constants/app_constants'
 import { get, parseJSON } from './auth'
 
 export const getTableauToken = () => get(`${CP_ANALYTICS_API}tableau/token`).then(parseJSON)
+export const getFilter = (filter) => get(`${CP_ANALYTICS_API}filter/${filter}`).then(parseJSON)
```
Comment (seddy) [inline app/js/services/cp_analytics_api_client.js]: This is wrong, it should be looking at client_portal-admin

File: app/js/services/tableau.js [hunk 2]
```diff
@@ -91,11 +91,21 @@ const getParameters = (workbook) => getWorkbookParameters(workbook).then((tablea
 
 /**
  * Returns formatted tableau filters as application filters
+ *
+ * This currently just fetches the filters from Tableau and trkrefs from current_user's profile
+ * It should also fetch the filters we need from the client_portal-analytics_backend in order to
+ * correctly select product names/etc.
```
Comment (HamishCharles) [inline app/js/services/tableau.js]: Is this comment the change we made or a further change that needs to be made? If it's what it does with this change the wording needs to be different.

Comment (seddy) [inline app/js/services/tableau.js]: Ah, good call - sorry, I put that in here before doing any actual development when I was trying to work out where the change goes :)!  Should be fine to delete and squash @ionut998

Comment (ionut998) [inline app/js/services/tableau.js]: I have changed the comment to reflect what the function currently does.

**Pull request merged**